airflow/providers/cncf/kubernetes/executors/kubernetes_executor.py

```diff
@@ -471,6 +471,15 @@ def _change_state(
         if TYPE_CHECKING:
             assert self.kube_scheduler
 
+        if state == TaskInstanceState.ADOPTED:
+            # When the task pod is adopted by another scheduler,
+            # then remove the task from the current scheduler running queue.
```

Comment:
Better to use `executor` instead of `scheduler`:
```suggestion
            # When the task pod is adopted by another executor,
            # then remove the task from the current executor running queue.
```